src/Stratis.Bitcoin.IntegrationTests/API/ApiSteps.cs

```diff
@@ -322,6 +321,17 @@ private void calling_getrawtransaction_verbose()
             this.send_api_get_request($"api/Node/getrawtransaction?trxid={this.transaction.GetHash().ToString()}&verbose=true");
         }
 
+        private void calling_getstakinginfo()
+        {
+            this.send_api_get_request("api/Staking/getstakinginfo");
```

Comment:
I wonder if we should make these a list of constants at the top of the file?

Reply:
Good idea. Done.